crates/matrix-sdk/src/widget/filter.rs

```diff
@@ -0,0 +1,241 @@
+#![allow(dead_code)] // temporary
```

Comment:
Still necessary?

Reply:
Yes, until the `matches` function actually starts getting used outside of tests.